waggle-dance-core/pom.xml

```diff
@@ -285,6 +286,16 @@
         <artifactId>spring-boot-maven-plugin</artifactId>
         <version>${spring-boot-maven-plugin.version}</version>
         <configuration>
+          <requiresUnpack>
+            <!-- We are hitting an issue introduced in hive-common-2.1.1 and fixed in https://issues.apache.org/jira/browse/HIVE-17155
```

Comment:
It won't break anything if we use Hive-3.0.0 so really it's up to the reality of the future. I would say the comment implies (by fixed) that it can be removed :)